lib/slack-ruby-bot-server/ext/slack-ruby-bot.rb

```diff
@@ -1,3 +1,3 @@
-%w(client commands/base).each do |ext|
-  require_relative "slack-ruby-bot/#{ext}"
-end
+require_relative 'slack-ruby-bot/client'
+require_relative 'activerecord/slack-ruby-bot/commands/base' if defined?(::ActiveRecord)
```

Comment:
This should be `require_relative "{SlackRubyBotServer::Config.database_adapter}/slack-ruby-but/commands/base"`.